gmm_client/src/client.erl

```diff
@@ -0,0 +1,105 @@
+%%%-------------------------------------------------------------------
+%%% @author Svatopluk
+%%% @copyright (C) 2021, <COMPANY>
+%%% @doc
+%%%
+%%% @end
+%%% Created : 10. maj 2021 16:54
+%%%-------------------------------------------------------------------
+-module(client).
+-author("Svatopluk").
```

Comment:
To było by ok, gdyby to nie była praca Inżynierska i gdyby to by twój login na githubie. Proponuję imię i nazwisko